health/health.d/dbengine.conf

```diff
@@ -23,4 +23,16 @@ lookup: sum -10m unaligned of I/O errors
   crit: $this > 0
  delay: down 1h multiplier 1.5 max 3h
   info: number of IO errors dbengine came across the last 10 minutes (CRC errors, out of space, bad disk etc)
-    to: sysadmin
\ No newline at end of file
+    to: sysadmin
+
+ alarm: 10min_dbengine_global_flushing_errors
+    on: netdata.dbengine_global_errors
+    os: linux freebsd macos
+ hosts: *
+lookup: sum -10m unaligned of flushing errors
+ units: errors
+ every: 3s
+  crit: $this > 0
+ delay: down 1h multiplier 1.5 max 3h
+  info: number of times dbengine failed to flush in time the last 10 minutes, metric data will not be stored in the database, please reduce disk load or use a faster disk
```

Comment:
Phrasing suggestion for this:

'number of times in the last 10 minutes that the dbengine failed to completely flush data to disk'

Flows a bit better from the perspective of a native AmE speaker and more concretely explains the exact conditions that resulted in the alarm.